dotcom-rendering/src/web/components/App.tsx

```diff
@@ -1136,7 +1136,6 @@ export const App = ({ CAPI, ophanRecord }: Props) => {
 					format={format}
 					discussionApiUrl={CAPI.config.discussionApiUrl}
```

Comment:
More for clarity, the reason we can't remove these from `App`, but have added them to `ArticleMeta`  is they are still depended on elsewhere in the code?

Reply:
Yes, exactly. They're still used by `Discussion`